### ISSUE TITLE
Finalizing

### ISSUE THREAD

**Issue by Cinecity** [Cinecity/portfolio#1]
@abhaysharma I can not figure out how to finalize this.
I have just structured it and i will add descriptions later.
i would like to know if i am going the right direction with it.

Many Thanks
Jay R.

**Comment by abhaysharma** [comment]
@Cinecity yes you are in the right direction. Follow the rubric to meet the requirements such as image optimization using picture tags, accessibility information, and Reead me file reflection, etc.
